staff.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Staff
+---
+
+## Identifying Staff
+
+The _staff_ are the volunteers who act as network and system administrators to
+keep Libera.Chat running smoothly.
+
+The complete current list of staff is available using the IRC command
+`/who libera/staff/*`.
+
+When this page was last updated, the staff members were:
+
+staff|staff|staff|staff
+--|--|--|--
+amdj     | emerson   | JonathanD  | nhandler
+bigpresh | emilsp    | jose       | nicole
+D[\_]     | Freso     | kline      | niko
+deedra   | Fuchs     | lausefuchs | njan
+edk      | glguy     | li         | Stx
+el       | jess      | mniip      | Swant
+
+Among the ops in `#libera` is also a service bot (which is not currently a
+member of staff):
+
+| bot
+|-
+| ozone
+
+## Contacting Staff
+
+### On IRC
+
+Assuming you can connect to IRC (`ircs://irc.libera.chat:6697`), that should be
```

Comment:
If anything, this should be a link like all others on the site. But there are also instances where you're able to connect to plaintext and not tls, and this encourages users with tls problems to seek help via slower methods (email) rather than trying plaintext.

Perhaps assuming you can connect (link to the guide on connecting)?

Reply:
I'll add a link

Reply:
In light of your comment below on focus (on identifying staff), I should just remove the ircs link entirely; and link `[connect to IRC](/somepage)` to a guide on connecting.